repos/system_upgrade/el7toel8/actors/pcidevicesscanner/libraries/pcidevicesscanner.py

```diff
@@ -61,7 +62,7 @@ def produce_pci_devices(producer, devices):
 def scan_pci_devices(producer):
     ''' Scan system PCI Devices '''
     try:
```

Comment:
Nit: you can use checked=False to suppress exception raise
```
output = run(['lspci', '-vmmk'], checked=False).get('stdout', [])
```

Reply:
Why would I want to supress that? As far as I know, `lspci` can't exit with a non-zero code.

Reply:
(Applied after a brief discussion.)